decred_wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -111,55 +118,48 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
         if self.validateSeed().valid {
             self.activateConfirmButton()
         } else {
-            self.deactivateConfirmButton()
+            self.deactivateConfirmButtonForError()
         }
     }
     
     func focusSeedWordInput(at tableRowIndex: Int) {
         let tableIndexPath = IndexPath(row: tableRowIndex, section: 0)
-        
         let nextSeedWordCell = self.tableView.cellForRow(at: tableIndexPath) as? RecoveryWalletSeedWordCell
         nextSeedWordCell?.seedWordAutoComplete.becomeFirstResponder()
         
-        self.tableView.scrollToRow(at: tableIndexPath, at: .middle, animated: true)
+        self.tableView.scrollToRow(at: tableIndexPath, at: .bottom, animated: true)
```

Comment:
What if you don't scroll? I mean, if you take off this line completely.

Reply:
Scrolling keeps the input textfield just below the keyboard input even if the user scrolls up and continue input.